yb-voyager/cmd/importData.go

```diff
@@ -1070,9 +1072,26 @@ func getYBSessionInitScript() []string {
 	return sessionVars
 }
 
+func removeExcludeTables(tableList []string) []string {
```

Comment:
You can take `excludeTableList []string` argument and then you can reuse this function in both export and import code paths.

Reply:
Agree to @amit-yb even can make this a general function `filterList` or some better name and put it inside the `utils` function, maybe reusable in future.